packages/page-democracy/src/Overview/ProposalCell.tsx

```diff
@@ -32,8 +32,8 @@ function ProposalCell ({ className = '', imageHash, proposal }: Props): React.Re
   // while we still have this endpoint, democracy will use it
   const democracy = api.query.democracy;
 
-  const displayProposal = democracy ? 
-    democracy.preimages
+  const displayProposal = democracy
+    ? democracy.preimages
```

Comment:
The `proposal` (which is from an `api.derive`) used the same underlying preimage logic as pre-defined here before the `const democracy = ...` introduction.

- if democracy.preimages, retrieve from there
- if preimage pallet, retrieve from there
- else go empty

That logic would align using simple `?.`. (Adjusted for optional preimage pallet as well)

```js
const displayProposal = api.query.democracy?.preimages
  ? proposal
  : preimage?.proposal;
```

Reply:
Thank you for the response!

The issue is that with using the `democracy?.` only the proposal hash is displayed. But with the code I changed it to, the motion displays properly. So `proposal` is not always undefined. 

I also found this ([here](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fpolkadot-rpc.dwellir.com#/council/motions))
![Screen Shot 2023-01-18 at 1 34 05 PM](https://user-images.githubusercontent.com/23270067/213289691-0dd44718-2bd2-4956-9557-5b6ab38b5e53.png)

The second row only displays the proposal hash. This is on Polkadot. The code I have does not work for this either. 

The following code does work to fix that issue on Polkadot, and the issue we are having.
```
  const displayProposal = api.query.democracy?.preimages
  ? proposal
  : preimage?.proposal 
    ? preimage.proposal
    : proposal;
```
(could probably be refactored, but first I want to check if that is a reasonable approach).

Here is what Polkadot looks like with that change:
![Screen Shot 2023-01-18 at 1 33 56 PM](https://user-images.githubusercontent.com/23270067/213290260-ba2e23c9-6733-4d00-ad5c-df31efbf4b5a.png)